### ISSUE TITLE
expose semi-core modules via resolvers

### ISSUE THREAD

**Issue by benmosher** [benmosher/eslint-plugin-import#479]
https://github.com/clayne11/eslint-import-resolver-meteor/issues/11

TL;DR: semi-magic modules (in this case, `meteor/*`) that exist on disk (and thus can be linted) but are baked into the environment (and thus, are not explicitly dependencies) ideally should not blow up `no-extraneous-dependencies`.

Hand-wavy possible solution: allow resolvers to define import types somehow. (maybe move the `importType` functionality down into the resolver spec?)

Or could keep the existing behavior when/if chosen resolver does not explicitly provide the designation.

**Comment by merlinpatt** [comment]
Any progress on this?

**Comment by jfmengels** [comment]
@merlinpatt No, no progress. If you feel like helping out, there could be some :)

**Comment by merlinpatt** [comment]
@jfmengels do you have suggestions on where to look to get started? I have 2 years dev experience but I've never written or contributed to a resolver before

**Comment by jfmengels** [comment]
Thanks for considering contributing! Unfortunately, that's a part I haven't touched either, and I don't want to send you on the wrong path.

@benmosher or @tleunen could probably advise you.

**Comment by benmosher** [comment]
@merlinpratt I appreciate your enthusiasm! I am not sure this is a good place for a first contribution, though, as it's mostly an API change. 

This is top of mind for me now that v2 is released, though, so I will be on it ASAP. 😁

**Comment by SrimalPriyanga** [comment]
Any progress to solve this issue?
